spec/features/circle_check_score_update_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe 'updating a circle check score' do 
+
+  before :each do 
+    create :circle_check_score
+  end
+
+
+  context 'with admin privilege' do 
+    it 'updates circle check score' do 
+      when_current_user_is :admin
+      visit circle_check_scores_url
+      fill_in 'circle_check_score_defects_found', with: '4'
+      click_on 'Save score'
+      expect(page).to have_text 'Score was saved.'
+      input = find_field :circle_check_score_defects_found
+      expect(input.value).to eql '4'
```

Comment:
Right, but this test could be a false positive if when you create it on line 6, it's initialized as 4. Then your test would pass because you're not actually changing the value.